src/components/graphql/fragments.js

```diff
@@ -3,7 +3,7 @@ import { graphql } from 'gatsby'
 export const backGroundBlur = graphql`
     fragment backGroundBlur on File {
         childImageSharp {
-            fluid(maxWidth: 1024, srcSetBreakpoints: [600, 1440]) {
+            fluid(quality: 90, maxWidth: 2048, srcSetBreakpoints: [600, 1440]) {
```

Comment:
This change affects all images that uses `backGroundBlur`. I suggest to specify the fix for the intended image. Or else check if all images that uses this property are okay with the change 🙏🏻